install/image-references

```diff
@@ -23,6 +23,10 @@ spec:
     from:
       kind: DockerImage
       name: registry.svc.ci.openshift.org/openshift:machine-os-content
+  - name: rhel-coreos
```

Comment:
I believe this needs to be `rhel-coreos-8` now.

Reply:
possibly bad form on my part -- I assume you're going commit-by-commit --  but I pulled that out when I added the extensions container, and then added them back in in the commit at the end (because in the old PR, that was what broke `images` when we didn't have the ART images, and I thought we might want to peel that one off)